### ISSUE TITLE
[iOS]UIGraphicsBeginImageContext() assertion. Use UIGraphicsImageRenderer to avoid this assert

### ISSUE THREAD

**Issue by robinmo** [appwoodpecker/woodpecker-client#2]
Xcode15 beta3
iOS17 beta4
The latest version of WoodPeckeriOS and mac app
![image](https://github.com/appwoodpecker/woodpecker-client/assets/6276950/4dc1b12a-87bc-4886-bfed-114e63b93ae8)

### Maybe the CGSizeZero judgement is needed.

1.integrate with WoodPeckeriOS
2.install mac WoodPecker app
3.run debug app and open mac WoodPecker view function
4.assertion occure.

the API says:
```
// UIImage context

// The following methods will only return a 8-bit per channel context in the DeviceRGB color space.
// Any new bitmap drawing code is encouraged to use UIGraphicsImageRenderer in lieu of this API.
UIKIT_EXTERN void     UIGraphicsBeginImageContext(CGSize size) API_DEPRECATED("Replace usage of UIGraphicsBeginImageContext with UIGraphicsImageRenderer.", ios(2.0, API_TO_BE_DEPRECATED), visionos(1.0, API_TO_BE_DEPRECATED));
```
it is the same as the issue: https://juejin.cn/post/7250288616534261797

**Comment by github-xiaogang** [comment]
Thanks, will fix it in a few days.

**Comment by github-xiaogang** [comment]
Hi robinmo, 
thanks for your feedback, the bug was fixed in 1.4.1, please update the WoodPeckeriOS framework later.
`pod 'WoodPeckeriOS', '1.4.1', :configurations => ['Debug']`

**Comment by robinmo** [comment]
Verify fixed on 1.4.1
